### PR TITLE
OC-225 ~ Implements generic JMS configuration using JmsConfigurationFactory.

### PR DESCRIPTION
**Pull request by kevinsmeets**

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://cla.opensmartgridplatform.org:80/OSGP/Protocol-Adapter-OSLP?pullRequest=89) <br/>All committers of the pull request should sign our Contributor License Agreement in order to get your pull request merged.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: kevinsmeets<br/>:x: MartijnSipsCGI

File: osgp-adapter-protocol-oslp-elster/src/main/resources/osgp-adapter-protocol-oslp-elster.properties [hunk 1]
```diff
@@ -72,62 +72,58 @@ jms.default.redelivery.delay=10000
...
 
-
 # =========================================================
-#  JMS Settings: Incoming Oslp Requests 
+#  JMS Settings: Incoming OSLP Requests 
 # =========================================================
 
 # --- DEFAULT DESTINATION ---
-jms.oslp.requests.queue=protocol-oslp-elster.1_0.osgp-core.1_0.requests
+jms.oslp.elster.requests.queue=protocol-oslp-elster.1_0.osgp-core.1_0.requests
```
Comment (kevinsmeets) [inline osgp-adapter-protocol-oslp-elster/src/main/resources/osgp-adapter-protocol-oslp-elster.properties]: Introduces unique properties for this component.

File: osgp-adapter-protocol-oslp/src/main/java/com/alliander/osgp/adapter/protocol/oslp/application/config/ApplicationContext.java [hunk 2]
```diff
@@ -31,12 +29,9 @@
 @Configuration
 @ComponentScan(basePackages = { "com.alliander.osgp.adapter.protocol.oslp", "com.alliander.osgp.core.db.api" })
 @EnableTransactionManagement()
-@Import({ MessagingConfig.class, OslpConfig.class, OslpPersistenceConfig.class, OsgpCoreDbApiPersistenceConfig.class })
```
Comment (kevinsmeets) [inline osgp-adapter-protocol-oslp/src/main/java/com/alliander/osgp/adapter/protocol/oslp/application/config/ApplicationContext.java]: Import statement is not needed.

File: signing-server/src/main/java/com/alliander/osgp/signing/server/application/config/ApplicationContext.java [hunk 3]
```diff
@@ -32,12 +31,9 @@
  */
 @Configuration
 @ComponentScan(basePackages = { "com.alliander.osgp.signing.server" })
-@Import({ MessagingConfig.class })
```
Comment (kevinsmeets) [inline signing-server/src/main/java/com/alliander/osgp/signing/server/application/config/ApplicationContext.java]: Import statement is not needed.

**Pull request merged**